AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -12,7 +12,7 @@
 
 #define NumberOfSections 10
 #define NumberOfRowsPerSection 20
-#define NumberOfReloadIterations 50
+#define NumberOfReloadIterations 250
```

Comment:
Yeah I def. agree once we have the issue I'm seeing fixed. I wanted to make sure you had something that would produce the problem ;) Let me tighten this up a bit and I'll update the stress test to make sure it can reproduce #566.